### ISSUE TITLE
[CLI] wandb.cli.cli.sweep name parameters requires a boolean

### ISSUE THREAD

**Issue by d3sm0** [wandb/client#2296]
**Description**
The default option for `--name` seems to be False, this cause the type of the name argument for the sweep cli command `--name` to be boolean. It appears that this might refuse any other non boolean type for the `--name` argument, for example a string.
 
https://github.com/wandb/client/blob/5a65037a435cbc8a885ab78fe5f23b8d7e10f5d2/wandb/cli/cli.py#L639

**Wandb features**
`wandb sweep`

**How to reproduce**
```
wandb sweep --name 'hello' sweep.yml
Usage: wandb sweep [OPTIONS] CONFIG_YAML_OR_SWEEP_ID
Try 'wandb sweep --help' for help.

Error: Invalid value for '--name': 'hello' is not a valid boolean.
```

**Environment**
- python 3.8.6
- wandb 0.10.32
- ubuntu

**Comment by tyomhak** [comment]
Hey @d3sm0 , I just tried the line below (with wandb 0.10.32), and it worked just fine:

`wandb sweep --name "sweep_name" config.yaml`

Can you try changing the single quotes around your sweep name to double quotes? Maybe that's causing the issue?

**Comment by RobertCsordas** [comment]
I have exactly the same issue. Changing the default=False to default=None solved it. I was running wandb 0.10.32, python 3.9, manjaro linux.

**Comment by tyomhak** [comment]
@RobertCsordas  you're right! Mine was click==7.1.2, and after upgrading to 8.0.1 I got the same error. 

I'm sharing this with the team, so we can get a fix for this.

**Comment by dannygoldstein** [comment]
This is fixed in 0.10.33

________________________________

**Comment by dannygoldstein** [comment]
Fix here: https://github.com/wandb/client/pull/2328
